examples/js/loaders/3MFLoader.js

```diff
@@ -1,3 +1,7 @@
+/**
+ * @author technohippy / https://github.com/technohippy
+ */
+
 THREE.ThreeMFLoader = function ( manager ) {
```

Comment:
Um, not sure this naming convention is intuitive :thinking:. Although `ThreeMFLoader` looks a bit strange, i think it is still a clearer name than `TMFLoader`...